CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## New Updates
 
+
+### 6.0 - _October 15, 2019_
```

Comment:
ambitious date :)

Reply:
I'll update the date once we get closer. Just a placeholder for now. Hopefully, we can push the beta soon.